src/asset_folder_importer/premiere_get_referenced_media/processor.py

```diff
@@ -198,8 +198,14 @@ def process_premiere_project(filepath, raven_client, vs_pathmap=None, db=None, c
     lg.debug("looking for referenced media....")
     for filepath in pp.getReferencedMedia():
         total_files += 1
-        lg.debug("Looking up {0}".format(filepath))
-        server_path = re.sub(u'^/Volumes', '/srv', filepath).encode('utf-8')
+        try:
```

Comment:
do we need try/except here? can we just use filepath.encode() regardless?

Reply:
That did not occur to me. It might work. I will test it.

Reply:
You would think that would work, but it does not. No idea why.